dvc/logger.py

```diff
@@ -101,14 +101,8 @@ def _description(self, message, exception):
         return description.format(message=message, exception=exception)
 
     def _walk_exc(self, exc_info):
-        import traceback
-
-        buffer = io.StringIO()
-
-        traceback.print_exception(*exc_info, file=buffer)
-
         exc = exc_info[1]
-        tb = buffer.getvalue()
+        tb = traceback.format_exc(*exc_info)
```

Comment:
Note that we no longer need `cause_tb` down below, so `tb_list` will only contain one entry, so we can get rid of it.